{
  "uid" : "e47a66fe2797915d",
  "name" : "reverts on transfer",
  "fullName" : "Contract: SafeERC20 with token that returns false on all calls.reverts on transfer",
  "historyId" : "1b98e7eea84e6e983a92798c98b528fe",
  "time" : {
    "start" : 1635555006423,
    "stop" : 1635555007399,
    "duration" : 976
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067b790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067b790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC20/utils/SafeERC20.test.js:21:80)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: SafeERC20 with token that returns false on all calls"
  }, {
    "name" : "testClass",
    "value" : "Contract: SafeERC20 with token that returns false on all calls"
  }, {
    "name" : "testMethod",
    "value" : "reverts on transfer"
  }, {
    "name" : "package",
    "value" : "Contract: SafeERC20 with token that returns false on all calls"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "392c0af092043f10",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0428430>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554941087,
        "stop" : 1635554942092,
        "duration" : 1005
      }
    }, {
      "uid" : "be34f0057af17b60",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042d1c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554921046,
        "stop" : 1635554921652,
        "duration" : 606
      }
    }, {
      "uid" : "f556cf0f087ba8cf",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067d9a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554909947,
        "stop" : 1635554910539,
        "duration" : 592
      }
    }, {
      "uid" : "94f916cc3639f1ab",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a066c3a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554905279,
        "stop" : 1635554905867,
        "duration" : 588
      }
    }, {
      "uid" : "5d0ef59ec5cc9dd6",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0435460>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554870054,
        "stop" : 1635554871021,
        "duration" : 967
      }
    }, {
      "uid" : "24a441ef27a41b01",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fb4c10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554841188,
        "stop" : 1635554841784,
        "duration" : 596
      }
    }, {
      "uid" : "b7d42732bc3141bf",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0673e50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554828320,
        "stop" : 1635554828909,
        "duration" : 589
      }
    }, {
      "uid" : "3c2dcd009e135a60",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04371c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554825253,
        "stop" : 1635554825842,
        "duration" : 589
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "e7e8966dea330b33",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/e7e8966dea330b33",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551255686,
          "stop" : 1635551255754,
          "duration" : 68
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "e47a66fe2797915d.json",
  "parameterValues" : [ ]
}